src/NServiceBus.Core/Settings/SettingsHolder.cs

```diff
@@ -5,7 +5,7 @@ namespace NServiceBus.Settings
     using System.Collections.Generic;
     using System.Configuration;
     using System.Linq.Expressions;
-    using NServiceBus.ObjectBuilder;
+    using global::NServiceBus.ObjectBuilder;
```

Comment:
Either global or just `using ObjectBuilder`. I like global better because it is more explicit.

Any thoughts?